operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/LocallyRunOperatorExtension.java

```diff
@@ -138,6 +144,17 @@ private static void applyCrd(InputStream is, String path, KubernetesClient clien
     }
   }
 
+  public static CustomResourceDefinition parseCrd(String path, KubernetesClient client) {
+    try (InputStream is = new FileInputStream(path)) {
+      return (CustomResourceDefinition) client.load(new ByteArrayInputStream(is.readAllBytes()))
```

Comment:
This should handle better weird cases: if there are more than just one resource in the file, if the CRD is not a v1 spec, if the resource in the file is not a CRD are the most obvious cases where this might fail.

Reply:
Update it to handle more CRD in file. I don't think we should support other CRD version that v1, note that this is not the resource definition version inside, CRD itself is stable for very long time.

Reply:
Yes, agree, we should fix that, thus not generate those anymore.